crates/precompile/src/kzg_point_evaluation.rs

```diff
@@ -45,7 +45,7 @@ fn run(input: &[u8], gas_limit: u64, env: &Env) -> PrecompileResult {
     let z = as_bytes32(&input[32..64]);
     let y = as_bytes32(&input[64..96]);
     let proof = as_bytes48(&input[144..192]);
-    if !verify_kzg_proof(commitment, z, y, proof, env.cfg.kzg_settings.get()) {
+    if KzgProof::verify_kzg_proof(commitment, z, y, proof, env.cfg.kzg_settings.get()).is_err() {
```

Comment:
What about when it returns false? That's why I had a separate function for it

Reply:
Will revert back

Reply:
Good and fast catch. commit: https://github.com/bluealloy/revm/pull/668/commits/eb8b2233713eda89949bb7ebbbfd2dae003d57ec